Packs/AWS-Enrichment-Remediation/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "AWS Enrichment and Remediation",
     "description": "Playbooks using multiple AWS content packs for enrichment and remediation purposes",
     "support": "xsoar",
-    "currentVersion": "1.1.10",
+    "currentVersion": "1.1.12",
```

Comment:
why jump straight to 1.1.12?

Reply:
version was jumped because this PR is dependent on https://github.com/demisto/content/pull/31958/files, need to bump again because now that one is 1.1.12

Reply:
[AWS E+R ver bump](https://github.com/demisto/content/pull/31951/commits/9f64c8eb2d32d00e116ba543ce86b1f4dd81281e)